src/core/lib/gprpp/arena.cc

```diff
@@ -58,7 +58,21 @@ Arena* Arena::Create(size_t initial_size) {
        GPR_CACHELINE_SIZE % GPR_MAX_ALIGNMENT == 0)
           ? GPR_CACHELINE_SIZE
           : GPR_MAX_ALIGNMENT;
-  return new (gpr_arena_malloc<alignment>(alloc_size)) Arena(initial_size);
+  return gpr_arena_malloc<alignment>(alloc_size);
+}
+
+Arena* Arena::Create(size_t initial_size) {
+  return new (ArenaStorage(initial_size)) Arena(initial_size);
+}
+
+std::pair<Arena*, void*> Arena::CreateWithAlloc(size_t initial_size,
```

Comment:
Please use `Pair<>` (defined in the `grpc_core` namespace) instead of `std::pair<>`.

Reply:
Done.